pkgs/development/tools/csvq/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "csvq";
+  version = "1.17.11";
+
+  src = fetchFromGitHub {
+    owner = "mithrandie";
+    repo = "csvq";
+    rev = "v${version}";
+    sha256 = "sha256-cH4uUi/QhOxf9aXh3wRozSKPTd13PnxydKFoG32MIxM=";
+  };
+
+  vendorSha256 = "sha256-BIa5vAEIDNdD+xW9z7geyBJnSbOeXxCl0aHIuAMBIgE=";
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "SQL-like query language for CSV";
+    homepage = "https://mithrandie.github.io/csvq/";
+    license = licenses.mit;
+    maintainers = with maintainers; [ tomodachi94 ];
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
I think we can drop this `platforms` attribute to use the default one of `buildGoModule`, it does not like there is something specific to the platform.

Reply:
Done.